src/main/java/org/stellar/sdk/responses/RootResponse.java

```diff
@@ -33,6 +39,88 @@ public class RootResponse extends Response {
   @SerializedName("current_protocol_version")
   int currentProtocolVersion;
 
+  @SerializedName("supported_protocol_version")
+  Long supportedProtocolVersion;
+
   @SerializedName("core_supported_protocol_version")
   int coreSupportedProtocolVersion;
+
+  @SerializedName("_links")
```

Comment:
for consideration, `Links` stands on it's own, could be worth pulling up to it's own class file rather than static under RootResponse.

Reply:
Many Response models have their own Links, and I think it's easier to maintain them by putting them together, just like what we did in stellar/go: https://github.com/stellar/go/blob/master/protocols/horizon/main.go#L225-L231